scripts/dataset.mdx

```diff
@@ -0,0 +1,5 @@
+<Block>
```

Comment:
Could you explain what the purpose of this file is?

From what I can tell, we'll generate a file in the form:

```
<yaml populated from json>
<This files contents unchanged?>
```

Does this filler text get changed elsewhere?